include/uv.h

```diff
@@ -1272,6 +1274,15 @@ UV_EXTERN int uv_os_gethostname(char* buffer, size_t* size);
 
 UV_EXTERN int uv_os_uname(uv_utsname_t* buffer);
 
+struct uv_metrics_s {
+  uint64_t loop_count;
+  uint64_t events;
+  uint64_t events_waiting;
+  /* private */
+  uint64_t* reserved[13];
```

Comment:
I wonder if we should make the reserved field a `void*`. It looks like other reserved fields in this file are `void*`.

Reply:
@cjihrig I was wondering that, but since the size of `void*` can differ depending on architecture I decided it would be better to use something that would give me a fixed number of bytes, but I can change it if you think it's more appropriate.